src/main/java/uk/ac/cam/cl/dtg/teaching/pottery/model/BuilderInfo.java

```diff
@@ -34,9 +32,8 @@
   public static final String STATUS_NOT_STARTED = "NOT_STARTED";
   public static final String STATUS_SCHEDULED = "SCHEDULED";
   public static final String STATUS_COPYING_FILES = "COPYING_FILES";
-  public static final String STATUS_COMPILING_TEST = "COMPILING_TEST";
-  public static final String STATUS_COMPILING_SOLUTION = "COMPILING_SOLUTION";
-  public static final String STATUS_TESTING_SOLUTION = "TESTING_SOLUTION";
+  public static final String STATUS_COMPILING_TESTS = "COMPILING_TESTS";
```

Comment:
Could you javadoc this one? (Am I right that this represents compiling the testing code within the task for the different variants?)

Reply:
Sure, will do.